.eslintignore

```diff
@@ -1,3 +1,4 @@
 # Ignore app templates from CI linting of stripes-cli due to template syntax
 # Run lint-templates to lint the app templates separately
 resources/**/*
+test
```

Comment:
We may want to think about a separate configuration for tests.  See what started here: I https://github.com/folio-org/stripes-cli/blob/master/test/.eslintrc